src/core/schedule.js

```diff
@@ -67,3 +43,35 @@ export const getFullSchedule = shiftWeek => {
 		})
 	return scheduleWeek.join('')
 }
+
+const getDayIndex = () => {
+	return {
+		today() {
+			const date = new Date()
+			const currentHours = date.getUTCHours()
+			// Time zone offset in Moscow
+			date.setUTCHours(currentHours + 3)
+			const currentDay = date.getDay()
+			const dayIndex = currentDay % 7
+			return dayIndex % 7
+		},
+		tomorrow() {
+			return this.today() + 1
+		},
+	}
+}
+
+const holidayStatus = () => {
+	const dayIndex = getDayIndex().today()
+	return orderedWeekDays[dayIndex] === 'Sunday'
+}
```

Comment:
1. If something is a boolean its name should start with a `is` or `has` prefix most of the time
2. Function name should start with a verb

I suggest `checkIsSunday`  for this function

Reply:
I'll fix it